common/changes/office-ui-fabric-react/messagebar-icon-aria_2019-05-16-22-35.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "[MessageBar] aria hidden for message bar icon",
```

nit: we usually use this format
```suggestion
      "comment": "MessageBar: aria hidden for message bar icon",
```